scripts/docker/hub/Dockerfile

```diff
@@ -16,16 +16,12 @@ RUN apt clean -y
 RUN rm -rf /tmp/* /var/tmp/* /var/lib/apt/lists/*
 
 RUN groupadd -g 1000 parity \
-  && useradd -m -u 1000 -g parity -s /bin/sh parity
+  && useradd -u 1000 -g parity -s /bin/sh parity
```

Comment:
I think it would still make sense to create the home directory, so that we can then create `/home/parity/.local/share/io.parity.ethereum/` in the Dockerfile. This would allow people to then mount a chain on the host into the container  without [permission issues](https://github.com/paritytech/parity-ethereum/pull/9689#issuecomment-429565718) and without having to pass `--base-path`.

TL;DR: I thnk we should leave the `-m` flag.